src/Traits/UrlTrait.php

```diff
@@ -33,6 +33,14 @@ protected function getFullUrl($url, ?string $locale = null)
 
     protected function stripApi($url)
     {
-        return preg_replace('#://api\.#', '://', $url);
+        $languages = LanguageProvider::getLanguageList();
+        foreach ($languages as $language) {
+            $parsedUrl = parse_url($url);
+            $parsedHomeUrl = parse_url($language->home_url);
+            if(stristr($parsedUrl['host'], $parsedHomeUrl['host']) !== false) {
```

Comment:
Du kan eventuelt bede om host med det samme:
```suggestion
            $parsedUrl = parse_url($url, PHP_URL_HOST);
            $parsedHomeUrl = parse_url($language->home_url, PHP_URL_HOST);
            if(stristr($parsedUrl, $parsedHomeUrl) !== false) {
```